resources/metadata/cnbBuild.yaml

```diff
@@ -1,6 +1,8 @@
 metadata:
   name: cnbBuild
-  description: Executes a Cloud Native Buildpacks build for creating a Docker container.
+  description: |
+    Executes a Cloud Native Buildpacks build for creating a Docker container. 
+    Please note, that the `cnbBuild` step currently is in beta state.
```

Comment:
I think we should convey two things:
1. The `cnbBuild` step generally works
1. The API might still change before we remove the *beta* notice

I'll think about something as a concrete suggestion, but feel free to propose something yourself.